lib/grant-manager.js

```
@@ -399,6 +399,10 @@ GrantManager.prototype.validateToken = function(token) {
   if ( token.content.iat < this.notBefore ) {
     return;
   }
+  
+  if ( token.signed == undefined ) {
```

Done. On that note, would it be useful to change the error handling to be more robust in general? I.e. exception handling instead of return-empty-on-error, and logging information as to WHY a token failed to validate?